metrics_sdk/lib/opentelemetry/sdk/metrics/aggregation/sum.rb

```diff
@@ -11,11 +11,12 @@ module Aggregation
         # Contains the implementation of the Sum aggregation
         # https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md#sum-aggregation
         class Sum
-          attr_reader :aggregation_temporality
+          attr_reader :aggregation_temporality, :is_monotonic
```

Comment:
@xuan-cao-swi typically I would think of these kinds of attributes hidden behind predicates. 

Would it not make sense to make the public interface of this attribute `monotonic?`

Reply:
The value in the protobuf is `is_monotonic`, so I think the goal was to match. I also found it confusing with Ruby's predicates.

https://github.com/open-telemetry/opentelemetry-ruby/blob/289b4aaa35f8b13803ab6c6c8b34fddd41525376/exporter/otlp/lib/opentelemetry/proto/metrics/v1/metrics_pb.rb#L11 

But, I believe some agents (Js?) just reduce this to `monotonic` in the SDK, so there's precedent to go the other way.

Reply:
Thanks for explaining that, though isn't this an internal Ruby Class and not generated from protobuf source?

Are we leaking the protobuf source and idioms into the SDK?

Reply:
Great point. I don't think we need to. I've used the protobuf as a guide before for implementation, but those are behind the scenes details that probably don't need to be propagated upstream. @garoazinha has updated the code to use the predicate style you suggested.